observatory_platform/dags/dummy_telescope.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Curtin University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Author: Aniek Roelofs
+
+from datetime import datetime
+from airflow import DAG
+from airflow.operators.dummy_operator import DummyOperator
+
+
+default_args = {
+    "owner": "airflow",
+    "start_date": datetime(2020, 8, 10)
+}
+
+
+with DAG(dag_id="dummy_telescope", schedule_interval="@daily", default_args=default_args, catchup=True) as dag:
+    telescope = DummyOperator(
+        task_id="telescope"
+    )
```

Comment:
Just add `queue='remote_queue'` to the DummyOperator so that when we use this DAG to test the VM create/destroy DAGs, the dummy_telescope DAG runs on the worker server.

Reply:
It seems that the queue parameter for `DummyOperator` gets ignored or it just runs on the default queue? So it  should be changed to the BashOperator, e.g. 

```
with DAG(dag_id="dummy_telescope", schedule_interval="@daily", default_args=default_args, catchup=True) as dag:
    task1 = BashOperator(
        task_id="task1",
        bash_command="echo 'hello'",
        queue='remote_queue'
    )

    task2 = BashOperator(
        task_id="task2",
        bash_command="echo 'world'",
        queue='remote_queue'
    )

    task1 >> task2
```